laghos_assembly.hpp

```diff
@@ -18,6 +18,8 @@
 #define MFEM_LAGHOS_ASSEMBLY
 
 #include "mfem.hpp"
+#include "general/forall.hpp"
+#include "linalg/dtensor.hpp"
```

Comment:
These paths (here and in a few other places) do not work when using an installed version of mfem.

One option is to add the flag `-I$(MFEM_DIR)/include/mfem` to the build options, if that directory exists.

As a longer term solution we probably need to add a new header `mfem-X.hpp` where `X` is something like `kernels`, `forall`, etc.

Reply:
I fixed this issue (for now) in df2dbed.

Reply:
Ok, thanks @v-dobrev, I sync the `serial` dir too with your changes.

Reply:
Ah, OK. I forgot about that directory. 😃